frontend/src/utils/templates.ts

```diff
@@ -59,13 +59,64 @@ const replacePlaceholders = (baseText: string, replacementConfig: { [key: string
   return newText;
 };
 
-export const confirmationTemplate = (replaceConfig: { [key: string]: string | string[] | undefined }) => {
+export const confirmationTemplateSubmission = (replaceConfig: { [key: string]: string | string[] | undefined }) => {
+  const PCNS_URL = 'https://pcns-prod-master.apps.silver.devops.gov.bc.ca';
+  const BC_BANNER = 'https://coms.api.gov.bc.ca/api/v1/object/446ee8ee-e302-4cb8-b44d-24a1f583edba';
+  const BC_FOOTER = 'https://coms.api.gov.bc.ca/api/v1/object/853de44a-e62f-41f5-81fd-6eff6cb66d52';
+
   const baseTemplate =
-    'Dear {{ contactName }},\n\n' +
-    'The confirmation ID for your submission is: {{ activityId }}.\n\n' +
+    '<div style="width: 880px">' +
+    '<img src="' +
+    BC_BANNER +
+    '" height="120rem" width="880px"  alt="B.C. Government Logo" /><br><br>' +
+    '<div style="margin-left: 3rem; margin-right: 3rem;">' +
+    'Dear {{ contactName }},<br><br>' +
+    '<b>Confirmation ID: {{ activityId }}</b><br><br>' +
     // eslint-disable-next-line max-len
-    'A Housing Navigator will review your submission and contact you. Keep the Confirmation ID for future reference.\n\n' +
-    'Regards,\n\n' +
-    'Permit Connect Navigator Service';
+    'Thank you for registering your project with the Navigator Service. We have successfully received your project submission. A Navigator will review your submission and contact you. Please keep your confirmation ID for future reference.<br><br>' +
+    'View your project submission <a href="' +
+    PCNS_URL +
+    '/housing/submission/intake?activityId={{ activityId }}&submissionId={{ submissionId }}">here<a>.<br><br>' +
+    'Regards,<br><br>' +
+    '<a href="' +
+    PCNS_URL +
+    '">Navigator Service<a><br><br><br>' +
+    '</div>' +
+    '<img src="' +
+    BC_FOOTER +
+    '" width="100%" alt="B.C. Government Footer" /><br><br>' +
+    '</div>';
+  return replacePlaceholders(baseTemplate, replaceConfig);
+};
+
+export const confirmationTemplateEnquiry = (replaceConfig: { [key: string]: string | string[] | undefined }) => {
+  const PCNS_URL = 'https://pcns-prod-master.apps.silver.devops.gov.bc.ca';
+  const BC_BANNER = 'https://coms.api.gov.bc.ca/api/v1/object/446ee8ee-e302-4cb8-b44d-24a1f583edba';
+  const BC_FOOTER = 'https://coms.api.gov.bc.ca/api/v1/object/853de44a-e62f-41f5-81fd-6eff6cb66d52';
```

Comment:
Should be a single set of constants in this file instead of duplicated in each function